### DISCUSSION TITLE
Is there a way to develop angular applications without having to build the application every time

### DISCUSSION THREAD

**Discussion by Levinus** [neutralinojs/neutralinojs#567] [Q&A]
As far as I understand,  currently you would have to `cd` into the angular project, build the project into the Neutralino resources folder and then run `neu run` in order to view the result in the neutralino window.

Currently I have just added a `package.json` in the root of the project with something like:
```
scripts: {
serve:"cd angular-frontend-folder && npm run build && cd .. && neu run"
}
```
But it would be nice if I wouldn't have to rebuild the angular code every time.

**Comment by kennycallado** [comment]
Hi,

Maybe you can try using `ng build  --watch` angular will be rebuilding every time you save.

Another way is with `ng serve`, you should change the url property of the neutralino.config.json to "http://localhost:4200" where the angular server is running by default. Then when you run `neu run` will display the aplication, however there is no neutralino api.

**Reply by Levinus** [reply]
Hi,

Thanks for your reply! 
I was looking for a solution that would provide the live reloading of `ng build --watch` or  `ng serve` with `neu run`/ the neutralino api. But I think this will, indeed, not be possible then.
